packages/data/addon/adapters/metadata/bard.ts

```diff
@@ -68,7 +68,7 @@ export default class BardMetadataAdapter extends EmberObject implements NaviMeta
 
   async fetchEverything(options?: MetadataOptions): Promise<TODO> {
     const fullViewReq = this.query('table', '', { query: { format: 'fullview' }, ...options });
-    const metricFunctionsReq = this.fetchAll('columnFunction');
+    const metricFunctionsReq = this.fetchAll('columnFunction', options);
```

Comment:
Come back to add a test?

Reply:
Yeah 👍

Reply:
https://github.com/yavin-dev/framework/issues/1403